Project.toml

```
@@ -1,16 +1,18 @@
 name = "DiffResults"
 uuid = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
-version = "1.0.3"
+version = "1.1.0"
 
 [deps]
-StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+StaticArraysCore = "1e83bf80-4336-4d27-bf5d-d5a4f845583c"
 
 [compat]
-StaticArrays = "0.8, 0.9, 0.10, 0.11, 0.12, 1.0"
-julia = "1"
+StaticArrays = "1.5.7"
```

Is this needed?

---

I guess it's essentially documentation, since the two packages are closely tied. 

This PR still needs https://github.com/JuliaArrays/StaticArrays.jl/pull/1094 :
```suggestion
StaticArrays = "1.5.8"
```